examples/tls/tls_client.c

```diff
@@ -235,14 +239,22 @@ int TPM2_TLS_ClientArgs(void* userCtx, int argc, char *argv[])
         if (rc != 0) goto exit;
     }
 #endif
+
     /* See if primary storage key already exists */
     rc = getPrimaryStoragekey(&dev, &storageKey,
         useECC ? TPM_ALG_ECC : TPM_ALG_RSA);
-    if (rc != 0) goto exit;
+    if (rc == 0) {
+        bindKey = &storageKey;
+    }
+    else {
+    #ifndef WOLFTPM_MFG_IDENTITY /* not fatal if using mfg identity */
```

Comment:
nitpick: if WOLFTPM_MFG_IDENTITY is defined and rc != 0, a warning/info print message could be helpful.